### ISSUE TITLE
install with webpack ?

### ISSUE THREAD

**Issue by imajim** [uxsolutions/bootstrap-datepicker#2644]
hello,

anyone install bootstrap datepicker with webpack ? i don's found a documentation

thx ^^

**Comment by rgyph** [comment]
Yes. You need to import the installed library into your script (and possibly locale) and you can use the datepicker as usual.

```
import 'bootstrap-datepicker';
import 'bootstrap-datepicker/js/locales/bootstrap-datepicker.ru';

$('.datepicker').datepicker();
```

**Comment by alexxandres** [comment]
CSS import is missing from the previous comment, for instance (I use "require" not "import" in my case but I guess it is similar):

`require('bootstrap-datepicker/dist/css/bootstrap-datepicker3.css');`
